static/app/views/starfish/modules/DBModule.tsx

```diff
@@ -27,7 +27,7 @@ export default function DBModule() {
           <Layout.Body>
             <Layout.Main fullWidth>
               <PageErrorAlert />
-              <PageFiltersContainer>
+              <PageFiltersContainer skipLoadLastUsedEnvironment>
```

Comment:
Maybe we should bring back a `StarfishPageFiltersContainer`, as I imagine we'll need this for FE and mobile stuff moving forward.

Reply:
Yeah that's a good point, I'll bring it back!